### PR TITLE
chore(frontend): Send bundlesize to zeus

### PR DESCRIPTION
**Pull request by MaxBittker**
this measures the gzipped size of vendor and app  and sends them to zeus as plaintext. probably not good enough to merge because I don't know the best way to make this parsable for zeus, but opening for discussion

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by benvinegar** [comment]
🆒

File: Makefile [hunk 1]
```diff
@@ -154,7 +154,7 @@ test-acceptance: build-platform-assets
 	py.test tests/acceptance --cov . --cov-report="xml:coverage.xml" --junit-xml="junit.xml" --html="pytest.html"
 	@echo ""
 
-lint: lint-python lint-js
+lint: lint-python lint-js measure-js-bundle
```
Comment (billyvg) [inline Makefile]: Is this going to work? Doesn't lint get run before test (which calls webpack)?

**Comment by billyvg** [comment]
Would be great for a v2 to have zeus accept a webpack profile dump `webpack --profile --json`  and have a webpack dashboard inside of zeus

**Comment by dcramer** [comment]
I’m fine with doing the profile stuff. Should be easy enough to build something that outputs useful information from it.

**Comment by dcramer** [comment]
I'm adding webpack stats support to Zeus real quick

File: Makefile (hunk 1 above)
Comment (MaxBittker) [reply Makefile]: i think you're right, good catch 😬

**Pull request closed without merging**